Lib/idlelib/colorizer.py

```diff
@@ -20,15 +20,16 @@ def make_pat():
     # self.file = open("file") :
     # 1st 'file' colorized normal, 2nd as builtin, 3rd as string
     builtin = r"([^.'\"\\#]\b|^)" + any("BUILTIN", builtinlist) + r"\b"
+    trailing_space = any("TRAILINGSPACE", [r"[ \t]+(?=\n)"])
```

Comment:
I am not sure if these are the only whitespace chars to catch.  Does not matter for now.